### PR TITLE
docs: add issue templates

### PR DESCRIPTION
**Pull request by aladdin-add**
it's mainly modified from eslint's issue templates. the repro link was changed to eslint-online-playground. A big thanks to @ota-meshi!

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-rule.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    description: What should the new rule do?
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: What type of rule is this?
+    options:
+      - Warns about a potential problem
+      - Suggests an alternate way of doing something
+      - Enforces a formatting/stylistic preference
```
Comment (scagood) [inline .github/ISSUE_TEMPLATE/new-rule.yml:23]: We should probably not offer stylistic rules anymore (I believe they're getting removed?)

**Review by scagood — COMMENTED** [review]
I like this idea 😁

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    label: Environment
+    description: |
+      Please tell us about how you're running ESLint.
+    value: |
+        Node version:
+        npm version:
+        ESLint version:
+        eslint-plugin-n version:
+        Operating System:
+  validations:
```
[the remark this answers is not in the document]
Comment (aladdin-add) [reply .github/ISSUE_TEMPLATE/bug-report.yml:21]: This section is all about env. I've added a new field "what rule do you want to report?".

File: .github/ISSUE_TEMPLATE/new-rule.yml (hunk 1 above)
Comment (aladdin-add) [reply .github/ISSUE_TEMPLATE/new-rule.yml]: stylistic rules will be deprecated in eslint v9.

yes, it's fine to remove it - the plugin does not offer stylistic rules.

**Review by scagood — APPROVED** [review]
Looks good to me :)

**Pull request merged**